### PR TITLE
SWAP-1306 As a user I want to see when my proposal has been scheduled

### PR DESCRIPTION
**Pull request by peter-asztalos**
## Description

This PR adds support for Users to see their upcoming (or all) beam times.

## How Has This Been Tested

- [x] manual
- [x] e2e test 

## Fixes

https://jira.esss.lu.se/browse/SWAP-1306

## Changes

- support to access scheduler
- core and scheduler github actions updated
- core and scheduler docker compose files updated
- API gateway updated

## Tests included/Docs Updated?

- [ ] I have added tests to cover my changes.
- [ ] All relevant doc has been updated

### CODE REVIEW

File: cypress/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM cypress/included:6.5.0
+FROM cypress/included:6.6.0
```
Comment (martin-trajanovski) [inline cypress/Dockerfile:1]: I see there is already version 6.8.0 but maybe we could update that next time to latest.

File: cypress.json [hunk 2]
```diff
@@ -1,4 +1,4 @@
 {
   "video": false,
-  "baseUrl": "http://127.0.0.1"
+  "baseUrl": "http://127.0.0.1:3000"
```
Comment (martin-trajanovski) [inline cypress.json:3]: I guess there wont be any problem to merge this. If so we do not need to change it locally.

File: package.json [hunk 3]
```diff
@@ -59,7 +59,7 @@
     "use-query-params": "^1.1.9",
     "yup": "~0.29.3"
   },
-  "proxy": "http://localhost:4000",
+  "proxy": "http://localhost:4100",
```
Comment (martin-trajanovski) [inline package.json:62]: Not sure why this change is needed from 4000 to 4100

**Review by martin-trajanovski — APPROVED** [review]
Otherwise it looks great!

File: package.json (hunk 3 above)
Comment (peter-asztalos) [reply package.json:62]: 4100 is the address of the api gateway

File: cypress/Dockerfile (hunk 1 above)
Comment (peter-asztalos) [reply cypress/Dockerfile:1]: Yes, unless there is any major issue with 6.6.0 we can upgrade next time.

**Review by jekabs-karklins — APPROVED** [review]
